tests/contrib/minikube_tests/test_kubernetes_pod_operator.py

```diff
@@ -0,0 +1,78 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Please use this header everywhere else as well. The other one is outdated and not accepted by Apache (anymore).